qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -223,12 +237,16 @@ def get_qubit_operators(
 
         # 2. Find the z2symmetries, set them in the qubit_converter, and apply the first step of the
         # tapering.
-        _, z2symmetries = self.qubit_converter.find_taper_op(
-            main_op, problem.symmetry_sector_locator
-        )
-        self.qubit_converter.force_match(z2symmetries=z2symmetries)
-        untap_main_op = self.qubit_converter.convert_clifford(main_op)
-        untap_aux_ops = self.qubit_converter.convert_clifford(aux_ops)
+        if isinstance(self.qubit_converter, QubitConverter):
+            _, z2symmetries = self.qubit_converter.find_taper_op(
+                main_op, problem.symmetry_sector_locator
+            )
+            self.qubit_converter.force_match(z2symmetries=z2symmetries)
+            untap_main_op = self.qubit_converter.convert_clifford(main_op)
+            untap_aux_ops = self.qubit_converter.convert_clifford(aux_ops)
+        else:
+            untap_main_op = main_op
+            untap_aux_ops = aux_ops
```

Comment:
Can we add a TODO comment here referring to https://github.com/Qiskit/qiskit-nature/issues/974?
Once we implement that mapper the logic here will need to be adapted to re-add the same two-step
logic that is currently only possible in the `QubitConverter` case.
The same might apply in other places below so it may be useful to add similar comments elsewhere.

Reply:
Yes, good point